static/js/login_or_signup.js

```diff
@@ -45,37 +46,40 @@ var checkKeypressIsSpace = function(event) {
 };
 
 $(function() {
-    var $signupForm = $('#signup-form');
-    var $loginForm = $('#login-form');
-
-    var submitHasBeenClicked = false;
-
     var displayNameField = 'displayName';
     var emailAddressField = 'emailAddress';
     var passwordField = 'password';
 
-    var signupFields = [
-        displayNameField,
-        emailAddressField,
-        passwordField,
-    ];
+    var signupFieldsMetaData = {
+        $form: $('#signup-form'),
+        fields: [
+            displayNameField,
+            emailAddressField,
+            passwordField,
+        ],
+        submitHasBeenClicked: false
```

Comment:
all key value pairs in a js object or array should be followed by a comma. The only exception is when an array or object is written on one line:
`var data = {value: 1};`